cybersecuritytools/aws_permissions_helper/aws_requests.py

```diff
@@ -0,0 +1,9 @@
+import re
+import sys
+
+pattern = "((?<=DEBUG: Request\s)(\w*\W\w*))"
+
+with open(sys.argv[1],"r") as data:
+    with open('aws_requests.txt','w') as aws:
```

Comment:
Deniz showed me a think where you create the 2 contexts and use a single `with` like 

```
terraform_log = open(sys.argv[1],"r") 
aws_request_log = open('aws_requests.txt','w')

with terraform_log, aws_requests_log: 
    ....
```

something like that anyway.